flamedisx/xenon/x1t_sr1.py

```diff
@@ -200,6 +204,16 @@ class SR1Source:
     variable_elife = True
     path_electron_lifetimes = ('1t_maps/electron_lifetimes_sr1.json',)
 
+    # Comsol map
+    variable_drift_field = True
+    path_drift_field = 'nt_maps/fieldmap_2D_B2d75n_C2d75n_G0d3p_A4d9p_T0d9n_PMTs1d3n_FSR0d65p.json'
+
+    # Field distortion map
+    path_drift_field_distortion = 'nt_maps/init_to_final_position_mapping_B2d75n_C2d75n_G0d3p_A4d9p_T0d9n_PMTs1d3n_FSR0d65p.json'
+
+    # FDC map
+    path_drift_field_distortion_correction = 'nt_maps/XnT_3D_FDC_xyt_MLP_v0.2_B2d75n_C2d75n_G0d3p_A4d9p_T0d9n_PMTs1d3n_FSR0d65p.json'
```

Comment:
This FDC is from nT and if you are to use a 1T source for an 1T analysis this would be wrong! Because you use it ...

Reply:
This is just a placeholder for the maps which the user should change.